doc/tutorials/virtualenvs.rst

```diff
@@ -0,0 +1,309 @@
+==========================================
+Creating and managing virtual environments
+==========================================
+
+When it comes to the management of third-party packages, Python
+has some complications:
+
+* By default, every third-party package will be installed to same directory.
+* Python is unable to differentiate between different versions of the same
+  package installed to that directory
+
+This means:
+
+* If you're working on a project and you make a backwards incompatible
+  change, then other projects that depend on it may be broken until you go
+  through and update them all with the necessary changes.
+* If you have two projects that depend on different version of the same
+  package then it becomes impossible for both to function simultaneously.
+
+In order to deal with these problems (and others) its recommended that you
```

Comment:
its -> it's

Reply:
You got the wrong one... this one had been correct after the last change, the one below is not changed...

Reply:
Whoops!

Reply:
OK I think they're all fixed properly